setup.py

```diff
@@ -6,6 +6,33 @@
 
 here = os.path.abspath(os.path.dirname(__file__))
 
+# Our basic list of 'extras'
+extras_requires = {
+    "schemas": ["jsonschema", "pydantic"],
+    "server": ["fastapi", "protobuf", "uvicorn"],
```

Comment:
as above, I'm leaning towards leaving protobuf as a required install to force serialization/de-serialization on all communication, particularly since it has zero dependencies

Reply:
OK, I'll move `protobuf` back to required